oak-segment-tar/src/test/java/org/apache/jackrabbit/oak/segment/standby/DataStoreTestBase.java

```diff
@@ -172,20 +175,20 @@ public void testResilientSync() throws Exception {
 
         // run 2: successful
         try (
-            StandbyServerSync serverSync = StandbyServerSync.builder()
-                .withPort(serverPort.getPort())
-                .withFileStore(primary)
-                .withBlobChunkSize(MB)
-                .build();
-            StandbyClientSync cl = StandbyClientSync.builder()
-                .withHost(getServerHost())
-                .withPort(serverPort.getPort())
-                .withFileStore(secondary)
-                .withSecureConnection(false)
-                .withReadTimeoutMs(4_000)
-                .withAutoClean(false)
-                .withSpoolFolder(spoolFolder)
-                .build()
+                StandbyServerSync serverSync = StandbyServerSync.builder()
+                        .withPort(serverPort.getPort())
+                        .withFileStore(primary)
+                        .withBlobChunkSize(MB)
+                        .build();
+                StandbyClientSync cl = StandbyClientSync.builder()
+                        .withHost(getServerHost())
+                        .withPort(serverPort.getPort())
+                        .withFileStore(secondary)
+                        .withSecureConnection(false)
+                        .withReadTimeoutMs(4_000)
+                        .withAutoClean(false)
+                        .withSpoolFolder(spoolFolder)
+                        .build()
```

Comment:
could you please avoid white-space only changes? (unless this aligns us with 1.8...)